src/content/docs/infrastructure/amazon-integrations/get-started/aws-integrations-metrics.mdx

```diff
@@ -9,10 +9,26 @@ redirects:
   - /docs/aws-metrics
 ---
 
-## Amazon Web Services Metrics [#aws-metrics-table]
-
 We recommend using the [AWS CloudWatch Metric Streams integration](/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream/) to ingest AWS services' metrics. For a complete list of available metrics, check the CloudWatch metrics listing documentation for each AWS service.  
 
+## Metrics naming convention [#metric-naming-convention]
+
+Metrics received from AWS CloudWatch are stored in New Relic as [dimensional metrics](/docs/telemetry-data-platform/ingest-manage-data/understand-data/new-relic-data-types/#dimensional-metrics) following this convention:
+  * Metrics are prefixed by the AWS namespace, all lowercase, where / is replaced with . :
+    * `AWS/EC2 -> aws.ec2`
+    * `AWS/ApplicationELB -> aws.applicationelb`
+  * The original AWS metric name with its original case:
+    * `aws.ec2.CPUUtilization`
+    * `aws.s3.5xxErrors`
+    * `aws.sns.NumberOfMessagesPublished`
+  * If the resource the metric belongs to has a specific namespace prefix, it is used. If the resource the metric belongs to doesn't have a specific namespace prefix, metrics use the `aws.` prefix.
+    * `aws.Region`
+    * `aws.s3.BucketName`
+
+Current namespaces supported by AWS can be found in the [CloudWatch documentation website](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html).
```

Comment:
Rather than passive voice here, maybe change to "For more information about namespaces supported by AWS, see  the [Amazon AWS CloudWatch documentation](link)."

Reply:
Good one! Thanks!